src/features/Settings/components/ExcludedAssets.tsx

```diff
@@ -0,0 +1,336 @@
+import React, { Dispatch, SetStateAction, useState, useContext } from 'react';
+import styled from 'styled-components';
+
+import { translateRaw } from '@translations';
+import {
+  TableCell,
+  TableRow,
+  EthAddress,
+  AssetIcon,
+  Currency,
+  DashboardPanel,
+  Typography,
+  Tooltip,
+  FixedSizeCollapsibleTable,
+  AddIcon
+} from '@components';
+import { BREAK_POINTS, COLORS, SPACING } from '@theme';
+import { Fiat, TUuid, IAccount } from '@types';
+import { truncate } from '@utils';
+import { SettingsContext } from '@services/Store';
+import { Balance, BalanceAccount } from '@features/Dashboard/components/WalletBreakdown/types';
+import { CenteredIconArrow } from '@components/IconArrow';
+
+const { SCREEN_MD } = BREAK_POINTS;
+
+const BalancesOnly = styled.div`
```

Comment:
We should be able to re-use components from the Dashboard for this and not re-implement it for settings? (Comment for the entire file in general)

Reply:
dun.